websites/D/Duolingo/dist/metadata.json

```diff
@@ -1,31 +1,51 @@
 {
   "$schema": "https://schemas.premid.app/metadata/1.5",
+  "service": "Duolingo",
   "author": {
-    "name": "TheSlowly",
-    "id": "505071508851523599"
+    "name": "Luke",
+    "id": "279622919242514432"
+  },
+  "category": "other",
+  "logo": "https://i.imgur.com/YKAf42L.png",
+  "thumbnail": "https://i.imgur.com/gVq8HiS.png",
```

Comment:
Use a screenshot of the homepage instead

Reply:
That screenshot is the homepage when you're logged in, would you still prefer the landing page for when you're not logged in instead? (https://i.imgur.com/M3nbjZJ.png)

Reply:
It is up to you, I'd rather set the thumbnail as the home page screenshot as the dashboard doesn't look as good on the store

Reply:
Alright that's fine, I'll change it.